control/canfd_control/MDK-ARM/startup_stm32h723xx.s

```diff
@@ -14,12 +14,23 @@
 ;******************************************************************************
 ;* @attention
 ;*
-;* Copyright (c) 2019 STMicroelectronics.
-;* All rights reserved.
+;* Copyright 2025 Reazon Holdings, Inc.
+;* Copyright 2024 STMicroelectronics.
 ;*
-;* This software is licensed under terms that can be found in the LICENSE file
-;* in the root directory of this software component.
-;* If no LICENSE file comes with this software, it is provided AS-IS.
+;* Licensed under the Apache License, Version 2.0 (the "License");
+;* you may not use this file except in compliance with the License.
+;* You may obtain a copy of the License at
+;*
+;*     http://www.apache.org/licenses/LICENSE-2.0
+;*
+;* Unless required by applicable law or agreed to in writing, software
+;* distributed under the License is distributed on an "AS IS" BASIS,
+;* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;* See the License for the specific language governing permissions and
+;* limitations under the License.
+;*
+;* Note that STMicroelectronics holds copyright only for the auto-generated
+;* code by STM32CubeMX.
```

Comment:
Is there any code we wrote in this file?
(Do we need to change this file?)

Reply:
There is no code we wrote in this file.
I think we don't have the way to write it by ourselves.

Reply:
Then we should not add our copyright.

Reply:
fix: [72a69d3](https://github.com/reazon-research/OpenArm/pull/119/commits/72a69d3b3204e2025cc78858acae28993263c3b9) Thanks. I removed our copyright here.